src/udp_wrap.cc

```diff
@@ -324,6 +328,42 @@ void UDPWrap::DropMembership(const FunctionCallbackInfo<Value>& args) {
   SetMembership(args, UV_LEAVE_GROUP);
 }
 
+void UDPWrap::SetSourceMembership(const FunctionCallbackInfo<Value>& args,
+                                  uv_membership membership) {
+  UDPWrap* wrap;
+  ASSIGN_OR_RETURN_UNWRAP(&wrap,
+                          args.Holder(),
+                          args.GetReturnValue().Set(UV_EBADF));
+
+  CHECK_EQ(args.Length(), 3);
+
+  node::Utf8Value sourceAddress(args.GetIsolate(), args[0]);
+  node::Utf8Value groupAddress(args.GetIsolate(), args[1]);
```

Comment:
Not that important but comon code style is `source_address`

Reply:
Fixed